README.md

```diff
@@ -41,6 +41,7 @@ jobs:
 | Name | Description | Required | Default |
 | ---     | ----------- | ----------- |----|
 | `manifest-path` | The relative path of the manifest file  | Required | - |
+| `stop-at-module` | Stop at the specified module, ignoring it and all the following ones. Note that using this option disables generating a bundle. | Optional | - |
```

Comment:
Nitpick: please remove `Note that` from the sentence. Please change ` generating a bundle` to ` generating bundles`.

Reply:
It would be also nice to document what is the default behavior to make the behavior difference pretty clear

Reply:
Done

Reply:
@tytan652 I guess you accidentally missed Bilal's comment above? :slightly_smiling_face:

Reply:
Yeah the page don't actualize if a message is added to conversation, sorry.

Reply:
How it is now ?

Reply:
If Bilal is happy with it, I'm happy with it too :+1: